docs/src/pages/comparing-astro-vs-other-tools.md

```diff
@@ -0,0 +1,169 @@
+
+Jekyll achieves this by pushing you to avoid JavaScript entirely. Jekyll sites are often written with little to no JavaScript at all, and instead promote server-side HTML rendering. This becomes an issue when you do need client-side JavaScript. It is up to you to create your own build pipeline for Jekyll. This can be time-consuming and forces you to set up bundling, minification, and other optimizations yourself.
+
+By contrast, Astro automatically builds your client-side JavaScript for you. Astro only sends the bare minimum amount of JavaScript to the browser, minified, bundled and optimized for production. While it is possible to achieve this yourself in Jekyll, with Astro this is built in by default.
+
+
+## Next.js vs. Astro
+
+[Next.js](https://nextjs.org/) is a popular website & application framework for React. 
+
+Next.js uses React for HTML templating. Astro lets you create pages using your favorite UI component libraries (React, Preact, Vue, Svelte, and others) or a built-in component syntax that is similar to HTML/JSX. 
+
+Both Next.js and Astro were designed to build any kind of website. Next.js does best with highly dynamic websites (like dashboards and inboxes) while Astro does best with highly static websites (like content and eCommerce websites).
+
+Next.js supports both Static Site Generation (SSG) and Server-Side Rendering (SSR). Today, Astro only supports Static Site Generation (SSG).
+
+#### Comparing Next.js vs. Astro Performance
+
+In almost all cases, Astro websites will load significantly faster than Next.js websites. This is because Astro automatically strips unnecessary JavaScript from the page, hydrating only the individual components that need it. This feature is called [partial hydration](/core-concepts/component-hydration).
+
+Next.js doesn't support partial hydration, and instead forces the user to load your entire JavaScript application in the browser. This creates a slower page load and worse performance for your website. Next.js has [experimental support](https://piccalil.li/blog/new-year-new-website/#heading-no-client-side-react-code) for fully-static, zero-JavaScript pages. However, there is no planned support for hydrating individual components on the page. This leaves you with an all-or-nothing limitation.
+
+Next.js has great built-in image optimizations, which could make Next.js a better choice for some image-heavy websites.
+
+#### Case Study: Building a Documentation Website
+
+[nextjs.org/docs](https://nextjs.org/docs/getting-started) is the official Next.js documentation website,  built with Next.js. The website offers a similar enough design and featureset to compare against the official Astro documentation website. This gives us a ***rough, real-world*** comparison between the two site builders for this common use-case. 
```

Comment:
This is great! I still agree that the lighthouse scores are a great proxy, and I think this shows off the improvements more without disregarding confounding variables :)